internal/app/cli.go

```diff
@@ -191,21 +191,21 @@ func (c *cli) parse() {
 }
 
 // readState gets the desired state from files
-func (c *cli) readState(s *state) {
+func (c *cli) readState(s *state) error {
 	// read the env file
 	if len(c.envFiles) == 0 {
 		if _, err := os.Stat(".env"); err == nil {
 			err = godotenv.Load()
 			if err != nil {
-				log.Fatal("Error loading .env file")
+				return fmt.Errorf("error loading .env file")
```

Comment:
Why not wrap and return the original error too? Same with L208, L230, L237, L243.